root/layout/components/Footer.js

```diff
@@ -1,95 +1,520 @@
+                  target="_blank"
+                >
+                  {l('#metabrainz')}
+                </a>
+              </li>
+              <li>
+                <span className="fs-5">
+                  {l('Discussion IRC:')}
                 </span>
-              ),
-            })}
-          </>
-        ) : null}
+                <a
+                  className="fw-bold fs-5 ms-1"
+                  href="https://kiwiirc.com/nextclient/irc.libera.chat/?#musicbrainz"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('#musicbrainz')}
+                </a>
+              </li>
+              <li>
+                <span className="fs-5">
+                  {l('Email:')}
+                </span>
+                <a
+                  className="fw-bold fs-5 ms-1"
+                  href="mailto:support@metabrainz.org"
+                >
+                  {l('support@metabrainz.org')}
+                </a>
+              </li>
+              <li>
+                <a
+                  href="https://twitter.com/MusicBrainz"
+                  rel="noreferrer"
+                  target="_blank"
+                >
+                  <FontAwesomeIcon
+                    icon={faTwitter}
+                    size="lg"
+                  />
+                </a>
+                <a
+                  className="ms-3"
+                  href="https://facebook.com/MusicBrainz-12390437194"
+                  rel="noreferrer"
+                  target="_blank"
+                >
+                  <FontAwesomeIcon
+                    icon={faFacebook}
+                    size="lg"
+                  />
+                </a>
+                <a
+                  className="ms-3"
+                  href="https://instagram.com/metabrainz"
+                  rel="noreferrer"
+                  target="_blank"
+                >
+                  <FontAwesomeIcon
+                    icon={faInstagram}
+                    size="lg"
+                  />
+                </a>
+                <a
+                  className="ms-3"
+                  href="https://linkedin.com/company/metabrainz"
+                  rel="noreferrer"
+                  target="_blank"
+                >
+                  <FontAwesomeIcon
+                    icon={faLinkedin}
+                    size="lg"
+                  />
+                </a>
+              </li>
+            </ul>
+            {serverLanguages && serverLanguages.length > 1 ? (
+              <LanguageMenu
+                $c={$c}
+                currentBCP47Language={
+                  $c.stash.current_language.replace('_', '-')
+                }
+                serverLanguages={serverLanguages}
+              />
+            ) : null}
+          </div>
+          <div className="col-sm-12 col-md-3">
+            <h3 className="fs-3 fw-bold color-black ms-3">
+              {l('Join Us')}
+            </h3>
```

Comment:
The list of documentation pages under this is not necessarily great, but we'd need to think carefully about which docs are good and should be linked, which are less good and should be linked but fixed, and which are not worth linking to. This can happen while this is in the branch.